src/components/CounterBadge/CounterBadge.st.css

```diff
@@ -0,0 +1,70 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, applyOpacity, fallback;
+}
+
+/*Overrides*/
+:vars {
+    /*
+        Background color
+        @default color-5 with opacity 0.1
+    */
+    CounterBadgeBgColor: --overridable;
+    /*
+        Text color
+        @default priority default and light: color-5, priority primary: color-1
+    */
+    CounterBadgeTextColor: --overridable;
+}
+
+:vars {
+    defaultBgColor: color(fallback(value(CounterBadgeBgColor), color(color-8)));
+    defaultTextColor: color(fallback(value(CounterBadgeTextColor), color(color-5)));
+}
+
+:vars {
+    lightBgColor: color(fallback(value(CounterBadgeBgColor), applyOpacity(color(color-8),0.1)));
```

Comment:
We have to check with Michal if the opacity should be added only on the default color or also on the color that is passed by the `CounterBadgeTextColor`.